src/Raven.Server/Config/Categories/DatabaseConfiguration.cs

```diff
@@ -119,5 +119,10 @@ public DatabaseConfiguration(bool forceUsing32BitsPager)
         [SizeUnit(SizeUnit.Megabytes)]
         [ConfigurationEntry("Databases.PulseReadTransactionLimitInMb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public Size PulseReadTransactionLimit { get; set; }
+
+        [Description("Whether to disable TCP data compression")]
+        [DefaultValue(false)]
+        [ConfigurationEntry("Databases.DisableTcpCompression", ConfigurationEntryScope.ServerWideOnly)]
+        public bool DisableTcpCompression { get; set; }
```

Comment:
`DatabaseConfiguration` isn't the suitable category anymore. Let's move this option to `ServerConfiguration` and name it `Server.Tcp.Compression.Disable`